packages/containers/examples/ExampleSidePanel.js

```diff
@@ -2,11 +2,30 @@ import React from 'react';
 import { IconsProvider } from '@talend/react-components';
 import { SidePanel } from '../src';
 
-export default function ExampleSidePanel() {
-	return (
+const ExampleSidePanel = {
+	'default settings': () => (
 		<div>
 			<IconsProvider />
 			<SidePanel actionIds={['menu:first', 'menu:second', 'menu:third']} />
 		</div>
-	);
-}
+	),
+	'injected settings': () => (
+		<div>
+			<IconsProvider />
+			<SidePanel
+				actionsComponents={[
```

Comment:
With the last changes, this seems outdated

Reply:
Yep sorry about that